utils/describe-ref/lib/describe-ref.js

```diff
@@ -10,6 +10,8 @@ module.exports.sync = sync;
 function getArgs(options, includeMergedTags) {
   let args = [
     "describe",
+    // return latest tag - otherwise the ref is incoorrect
```

Comment:
By "correct" you mean... detects broken tags that aren't properly annotated? There should be no situation in which Lerna itself creates release tags that are undetectable _without_ the `--tags` option here.

Reply:
I have met same issue.

I've release packages and tagged git tag via GitHub API.
The tag is lightweight(non-annotated) tag.

- https://git-scm.com/docs/git-describe

`git descript` can not get lightweight(non-annotated) tag without `--tags` flag.
This commit will fix this issue that I met.

Example repository that used  GitHub API for tagging `git tag`

https://github.com/azu/lerna-monorepo-github-actions-release

```
❯ git describe
v1.3.0-68-ge149829

❯ git describe --tags
v2.0.1-3-ge149829
```